docs/project/get-help.md

```diff
@@ -92,62 +97,122 @@ the easiest way to connect to IRC.
 
-    The system connects you to chat. You'll see a lot of text. At the bottom of
-    the display is a command line. Just above the command line the system asks 
-    you to register.
+        /msg NickServ VERIFY REGISTER yournickname somecode
 
-    ![Login screen](/project/images/irc_after_login.png)
+Your nickname is now registered to chat on freenode.net.
 
+[Jump ahead to join a docker channel and start chatting](#join-a-docker-channel)  
 
-4. In the command line, register your nickname.
+## IRCCloud
 
-        /msg NickServ REGISTER password youremail@example.com
+IRCCloud is a Web Based IRC client service that is hosted in the cloud. 
 
-    ![Login screen](/project/images/register_nic.png)
+> **Note:** 
+> IRCCloud is a Freemium product. 
 
-    The IRC system sends an email to the address you
-    enter. The email contains instructions for completing your registration.
+To begin the registration process for IRCCloud select the following link:
+  <a href="https://www.irccloud.com/invite?channel=%23docker&amp;hostname=chat.freenode.net&amp;port=6697" target="_blank">Join the #docker channel on chat.freenode.net</a>
 
-5. Open your mail client and look for the email.
+The following web page is displayed in your browser:
 
-    ![Login screen](/project/images/register_email.png)
+ ![IRCCloud Register screen](/project/images/irccloud-join.png)
+  
+If this is your first time using IRCCloud enter a valid email address in the
+form; check your email for an invite from IRCCloud; and follow the instructions
+provided in the email. People who have already registered with IRCCloud can
+select the "sign in here" link. Additionally, people who are already registered
+with IRCCloud may have a cookie stored on their web browser that enables a quick
+start "let's go" link to be shown instead of the above form. In this case just
+select the "let's go" link and [jump ahead to start chatting](#start-chatting)
 
-6. Back in the browser, complete the registration according to the email.
+After following the instructions in your email you should have an IRCCloud
+Client web page in your browser:
 
-        /msg NickServ VERIFY REGISTER moxiegirl_ acljtppywjnr
+ ![IRCCloud](/project/images/irccloud-register-nick.png)
 
-7. Join the `#docker` group using the following command.
+The message shown above may appear indicating that you need to register your
+nickname. 
+
+To register your nickname enter the following message into the command line bar
+at the bottom of the IRCCloud Client:
+        
+        /msg NickServ REGISTER yourpassword youremail@example.com
+
+This command line bar is for chatting and entering in IRC commands.
+
+Check your email for an invite to freenode.net:
+
+ ![Login screen](/project/images/register_email.png)
+
+Back in the browser, complete the registration according to the email.
+
+        /msg NickServ VERIFY REGISTER yournickname somecode
+
+## Join a Docker Channel
```

Comment:
## Tips
The procedures in this section apply to both IRC clients.   
### Set a nickname
### Join a Docker channel
### Start chatting